boardgame-ui/src/utils/api/Game.js

```diff
@@ -74,10 +79,15 @@ export default class Game extends Model {
     if(this.isNew()) {
       return super.save();
     }
-    else {
+    else if (this.id_event != null) {
       let {data} = await axios.put(`/event/${this.id_event}/${this.className}/${this.id}`, this.getPublicProperties());
       this.populate(data);
       return this;
     }
+    else {
+      let {data} = await axios.put(`/game/${this.id}`, this.getPublicProperties());
```

Comment:
Got 404 error on this endpoint.

Reply:
Should be fixed on develop. BGWE/BoardgameWE@7b3908c